pages/now-showing.js

```diff
@@ -50,18 +50,22 @@ NowShowing.Chat = styled.div`
 `;
 
 NowShowing.getInitialProps = async () => {
-  if (process.env.NODE_ENV !== 'production') {
-    return { isLive: true };
-  }
+  let isStreamLive;
```

Comment:
This function makes sense overall, but one thing I'm wondering about is what we actually want to do if we fail to get the isLive and schedule down from Contentful. Currently, it looks like these values would return with prop values of undefined. Should we intentionally throw an error here? Alternatively, do we want to have some default/backup state so that things can fail more gracefully (though this could be pushed back to a later PR)?

Reply:
Yeah, you're right this would be smart! Will open a ticket for this and handle in a separate PR

Reply:
Gotcha. In the meantime, I might suggest throwing an error here that has a descriptive name so that it’s easier to figure out the cause if it happens